charts/portal/templates/cronjob-backend-processes.yaml

```diff
@@ -295,6 +295,23 @@ spec:
                 secretKeyRef:
                   name: "{{ .Values.backend.interfaces.secret }}"
                   key: "bpndidresolver-api-key"
```

Comment:
Please also remove the apiKey as it will be removed with https://github.com/eclipse-tractusx/portal-backend/pull/1129

Reply:
bpnDidResolver -> apiKey has been removed